### DISCUSSION TITLE
Parsing continuous time-series CLS file format

### DISCUSSION THREAD

**Discussion by kebonly** [zqfang/GSEApy#202] [Ideas]
gsea_cls_parser in the parser module assumes that the *.cls file describes a **categorical** class file format. According to the [Broad Institute](http://software.broadinstitute.org/cancer/software/gsea/wiki/index.php/Data_formats#CLS:_Categorical_.28e.g_tumor_vs_normal.29_class_file_format_.28.2A.cls.29), *.cls files can also describe **continuous time-series** file format.

It would be nice if the gsea_cls_parser could parse this continuous time-series format given the specifications described by the Broad Institute.

Thank you!

**Comment by zqfang** [comment]
Hi @lykevin99 , it's not supported yet. 

However, you could write your own parser, since it's a simple text file.
